input/write_reactors.py

```diff
@@ -42,15 +42,83 @@ def read_csv(csv_file):
 
     """
 
-    reactor_lists = np.genfromtxt(csv_file,
+    reactor_lists = np.genfromtxt(csv_file, skip_header = 1,
                                   delimiter=',',
-                                  dtype=('S128', 'S128', 'int', 'int', 'int'),
-                                  names=('country', 'reactor_name', 'capacity',
-                                         'n_assem_core', 'n_assem_batch'))
+                                  dtype=('S128', 'S128', 'S128', 'int', 
+                                         'int', 'int', 'S128',
+                                         'S128', 'int', 'int', 'int',
+                                         'int', 'int', 'int',
+                                         'int', 'int', 'float'),
+                                  names=('country', 'reactor_name', 'type', 'capacity',
+                                         'n_assem_core', 'n_assem_batch','status',
+                                         'operator', 'const_date', 'cons_year', 'first_crit',
+                                         'entry_time', 'lifetime', 'first_grid',
+                                         'commercial', 'shutdown_date', 'ucf'))
 
     return reactor_lists
 
 
+def convert_dates(init_date, start_date, end_date):
```

Comment:
Generally speaking, this function does too many things at once. There should be a function that converts the date format to y, m, d. That function should be separate from the function that determines the lifetime from the start and end dates, the function that converts start and end into simulation style dates (adjusted for the sim_start).

I'd strongly recommend small simple functions, so you don't repeat yourself (the DRY principle, again, please see http://journals.plos.org/plosbiology/article/file?id=10.1371/journal.pbio.1001745&type=printable)

- get_ymd(yyyymmdd) : could return the three-member tuple (y, m, d)
- get_lifetime(start, end) : start & end could each be three-member tuples, (y, m, d)
- convert_to_sim_time(raw_time, sim_start):  raw_time and sim_start could each be three-member tuples, (y, m, d). return sim_start + raw_time